packages/ckeditor5-utils/src/focustracker.ts

```diff
@@ -136,3 +269,20 @@ export default class FocusTracker extends /* #__PURE__ */ DomEmitterMixin( /* #_
 		}, 0 );
 	}
 }
+
+/**
+ * A {@link module:ui/view~View} instance with a {@link module:utils/focustracker~FocusTracker} instance exposed
+ * at the `#focusTracker` property.
+ */
+export type ViewWithFocusTracker = View & { focusTracker: FocusTracker };
+
+/**
+ * Checks whether a view is an instance of {@link ~ViewWithFocusTracker}.
+ */
+export function isViewWithFocusTracker( view: any ): view is ViewWithFocusTracker {
+	return 'focusTracker' in view && view.focusTracker instanceof FocusTracker;
+}
+
+function isElement( value: any ): value is Element {
```

Comment:
I didn't want to pollute this PR with unrelated changes, but it seems we have several similar functions scattered in the codebase. I'll create a follow-up and move this to `utils/src/dom` to share across the project.